src/main/webapp/javascript/tests/localUploader_test.js

```diff
@@ -0,0 +1,22 @@
+QUnit.module('filesChangedSetFilename', () => {
```

Comment:
I attempted to add some tests for the updated filename parsing function here, based on existing QUnit tests, but it seems that all of these tests in the `javascript/tests` directory are currently failing. Not sure if these are still being maintained. Let me know how I should proceed here!

Reply:
aha, that was the bit I missed for testing! Great catch! We have new tests that _are_ are being maintained. backend are here `/src/test` and frontend are here `/frontend/src/__tests__`. Making a note to get test locations documented and we need to pull out the old tests so this doesn't happen again.